src/main/java/net/minecraftforge/fml/common/FMLCommonHandler.java

```diff
@@ -228,7 +229,7 @@ public void computeBranding()
             }
             int tModCount = Loader.instance().getModList().size();
             int aModCount = Loader.instance().getActiveModList().size();
-            brd.add(String.format("%d mod%s loaded, %d mod%s active", tModCount, tModCount!=1 ? "s" :"", aModCount, aModCount!=1 ? "s" :"" ));
+            brd.add(StatCollector.translateToLocalFormatted("fml.menu.modsLoadedActiveSingular", tModCount, tModCount!=1 ? StatCollector.translateToLocalFormatted("fml.menu.modsLoadedPlural") :"", aModCount, aModCount!=1 ? StatCollector.translateToLocalFormatted("fml.menu.modsActivePlural") :"" ));
             brandings = brd.build();
```

Comment:
I would avoid anything in the main menu like this, it gives people a vector for vanity/annoyance to the end user by allowing them to clutter the main menu. We already give them 1 line worth of branding don't want to open up more.